zoo_frontend/src/static/Users.js

```diff
@@ -0,0 +1,22 @@
+import axios from 'axios';
```

Comment:
If we are going to follow this pattern, we probably wont to move these files under src/api/... because these files arent technically static.